packages/iql/package.json

```diff
@@ -45,7 +45,8 @@
     },
     "devDependencies": {
         "light-server": "^2.9.1",
-        "peggy": "^4.0.2"
+        "peggy": "^4.0.2",
+        "webpack": "^5.90.3"
```

Comment:
Should be removed. We have webpack as a dep in the root project.

Reply:
It's still a dev dependency for the iql package.

Reply:
it isnt as we track such "global stuff" in the root package json, remove it please.